assignment4/ques11.js

```diff
@@ -0,0 +1,18 @@
+// Write a program to sort an array (bubble sort)
+
+let inputArray = [7, 4, 2, 1, 9];
+
+function bubbleSort(arr) {
+    for (let i = arr.length - 1; i > 0; i--) {
```

Comment:
What if provided array is already sorted? all the loops will still itterate any possible solution for that?